antenna/throttler.py

```diff
@@ -90,7 +106,7 @@ def throttle(self, raw_crash):
 
         """
         for rule in self.rule_set:
-            match = rule.match(raw_crash)
+            match = rule.match(self, raw_crash)
```

Comment:
In order for rule conditions to use configuration, I needed to pass the configuration in as an argument. Instead of doing that, I passed the throttler. I figure this lets conditions do other things, too.

Further, I decided this reads poorly:

```
lambda config, x: ...
```

Config? What config? There's lots of config everywhere.

Instead, it's this:

```
lambda throttler, x: throttler.config('something')
```

That feels a lot clearer to me.